From 32dba1f82848e2bf4e2546dac06296fa7b387dff Mon Sep 17 00:00:00 2001
From: Rob Boehne <robb@datalogics.com>
Date: Mon, 16 Nov 2020 09:06:42 -0600
Subject: [PATCH] Check for HAVE_SYS_SYSCALL_H

Fix a compilation error on systems without sys/syscall.h by applying the same checks used elsewhere.
---
 randutils.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/randutils.c b/randutils.c
index 80893d3..8bf3af6 100644
--- a/randutils.c
+++ b/randutils.c
@@ -11,9 +11,14 @@
 #include <fcntl.h>
 #include <stdlib.h>
 #include <string.h>
+
+#if defined(HAVE_SYS_TIME_H)
 #include <sys/time.h>
+#endif
 
+#if defined(__linux__) && defined(HAVE_SYS_SYSCALL_H)
 #include <sys/syscall.h>
+#endif
 
 #include "randutils.h"
 
-- 
2.15.2

